internal/cli/helpers.go

```diff
@@ -3,22 +3,28 @@ package cli
 import (
 	"fmt"
 	"path/filepath"
-	"regexp"
 	"strings"
 
 	"k8s.io/client-go/util/homedir"
 )
 
+const (
+	DeviceKind            = "device"
+	EnrollmentRequestKind = "enrollmentrequest"
+	FleetKind             = "fleet"
+	RepositoryKind        = "repository"
+	ResourceSyncKind      = "resourcesync"
```

Comment:
+ 1

Reply:
question: are these globally used or is this just cli

Reply:
Just CLI